dcs_deploy.py

```diff
@@ -235,6 +333,12 @@ def check_optional_arguments(self):
             print("Exitting!")
             exit(6)
 
+        if self.args.rootfs is not None and self.args.app_size is None:
+            print('''
+                  WARNING! You did not specify --app_size parameter. 
+                  You may get 'No space left on device' error while flashing custom rootfs.
+                  ''')
+
```

Comment:
We can consider adding action from the user here, to terminate, or continue with the flash.